modules/mw.cx.TargetArticle.js

```
@@ -216,13 +222,47 @@ mw.cx.TargetArticle.prototype.publishSection = function () {
+	console.log("publishResult:");
+
+	if (publishResult.save_result_all) {
+		console.log("_____");
+		console.log("local result: " + JSON.stringify(publishResult.save_result_all.result));
+		console.log("mdwiki_result: " + JSON.stringify(publishResult.save_result_all.mdwiki_result));
+	} else {
+		console.log(JSON.stringify(publishResult));
+	}
+	// {"result":"error","edit":{"error":"noaccess","username":"Mr. Ibrahem"}}
 
 	if (publishResult.result === 'success') {
```

ESLint

[error] There must be a space after this paren.

(space-in-parens)